src/osd/strconv.h

```diff
@@ -25,22 +25,30 @@
 
 #include <windows.h>
 
-// the result of these functions has to be released with osd_free()
-
+std::string astring_from_utf8(const std::string &s);
 std::string astring_from_utf8(const char *s);
+std::string &astring_from_utf8(std::string &dst, std::string &s);
```

Comment:
What's the actual purpose of this overload?  It turns a `std::string` into another std::string?  The whole `astring_from_utf8` is leftover crud from before the `astring` to `std::string` migration, and should probably be removed unless you can come up with a good justification for it.  The MAME `astring` was always UTF-8, just like the`std::string` objects we pass around internally are always UTF-8.